source/release-notes.txt

```diff
@@ -11,6 +11,32 @@ Release Notes
    :class: twocols
 
 
+|compass| 1.42.0
+----------------
+
+*Released January 31, 2024*
+
+New Features
+
+- Compass now supports :ref:`Bulk Update <compass-bulk-update>` and 
+  :ref:`Bulk Delete<compass-bulk-delete>`
+  operations (:issue:`COMPASS-7329`, :issue:`COMPASS-7330`).
+
+Bug Fixes
+
+- Fixed namespace stats that refresh after document updates.
+- Fixed table card autosizing (:issue:`COMPASS-7548`).
+- Fixed an issue when opening a new collection tab when an existing 
+  collection tab with the same name was already open 
+  (:issue:`COMPASS-7556`).
+- Fixed an issue when switching tabs would reset vertical scrolling 
```

Comment:
[nit suggestion] I feel like this should be "where" instead of "when"
```suggestion
- Fixed an issue where switching tabs would reset vertical scrolling 
```